tools/grammars/compiler/converter.go

```diff
@@ -248,13 +257,25 @@ func (conv *Converter) Report() error {
 	return nil
 }
 
+var VERSION_RE *regexp.Regexp = regexp.MustCompile("VERSION = \"(.+)\"")
```

Comment:
Might be a little cleaner to change `version.rb` to do `VERSION = File.read("#{__dir__}/VERSION")` to simplify this, but this is clever and works 👍

Reply:
I prefer your suggestion 100%. Thank you!